Bugzilla/App/Plugin/OAuth2/Provider.pm

```diff
@@ -111,11 +113,15 @@ sub _resource_owner_confirm_scopes {
     return undef;
   }
 
+  # Deny access if hostname of redirect_uri doesn't match
+  # the hostname assigned to the client id
```

Comment:
Can you add this comment to the function definition?

Reply:
Will do.